Unable to reference Serilog in Full .NET Framework project

I'm working on a full .NET Framework Windows Service and trying to reference the SeriLog extension. When I do I get the following error:

```
Failed to add reference to 'System.Runtime'. Please make sure that it is in the Global Assembly Cache.
```

I believe it may be because of the following in the `project.json` file of this repo.

```
"net4.5": {
  "frameworkAssemblies": {
    "System.Runtime": ""
  }
}
```

If you add `System.Runtime` as a straight dependency instead of as a `frameworkAssemblies` entry this should then work as expected

Thanks for the report James. After some trial and error, I've managed to get this building with a root dependency of:

```json
    "System.Runtime": "4.1.0-rc2-24027"
```

But, we end up with a warning:

```
Compiling Serilog.Extensions.Logging for .NETFramework,Version=v4.5
C:\Development\Open Source\nblumhardt\serilog-extensions-logging\src\Serilog.Extensions.Logging\project.json(14,40): warning NU1007: Dependency specified was System.Runtime >= 4.1.0-rc2-24027 but ended up with System.Runtime 4.0.0.0.
```

Unfortunately, pretty much any other permutation fails to build entirely, including setting an explicit 4.0.0 version dependency in the .NET 4 project.

Do you have any insights into this?

--- a/samples/Sample/project.json
+++ b/samples/Sample/project.json
@@ -5,15 +5,12 @@
   },
   "dependencies": {
     "Serilog.Extensions.Logging": { "target": "project" },
-    "Serilog.Sinks.Literate": "2.0.0-rc-25"
+    "Serilog.Sinks.Literate": "2.0.0-rc-25",
+    "Microsoft.Extensions.Logging": "1.0.0-rc2-final"
   },
 
   "frameworks": {
-    "net4.6": {
-      "frameworkAssemblies": {
-        "System.Runtime": ""
-      }
-    },
+    "net4.6": {},
     "netcoreapp1.0": {
       "define": [ "ASYNCLOCAL" ],
       "dependencies": {

--- a/src/Serilog.Extensions.Logging/project.json
+++ b/src/Serilog.Extensions.Logging/project.json
@@ -9,7 +9,7 @@
     "iconUrl": "http://serilog.net/images/serilog-extension-nuget.png"
   },
   "dependencies": {
-    "Microsoft.Extensions.Logging": "1.0.0-rc2-final",
+    "Microsoft.Extensions.Logging.Abstractions": "1.0.0-rc2-final",
     "Serilog": "2.0.0-rc-576"
   },
   "buildOptions": {
@@ -19,9 +19,7 @@
   },
   "frameworks": {
     "net4.5": {
-      "frameworkAssemblies": {
-        "System.Runtime": ""
-      }
+      "dependencies": { "System.Runtime": "4.0.0" }
     },
     "netstandard1.3": {
       "buildOptions": {

